DIPs/DIP-58.md

```diff
@@ -0,0 +1,53 @@
+---
+DIP: 55
+Title: Story of Devcon
+Status: Draft
+Themes: Developer community, Content production, Community Involvement, Social
+tags:
+  - Communications
+Instances:
+  - Devcon8
+Authors: peth@metagame.wtf, eowulia@metagame.wtf
+Resources Required: Physical space at venue, Extra equipment or smol budget to rent it, permission to use past content
```

Comment:
```suggestion
```
let's put everything in "# Operational Requirements & Ownership" - and be very specific.